presto-native-execution/pom.xml

```diff
@@ -299,5 +310,86 @@
+                                            <directory>${project.parent.basedir}/presto-server/target</directory>
+                                            <includes>
+                                                <include>presto-server-*.tar.gz</include>
+                                            </includes>
+                                        </resource>
+                                    </resources>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>com.spotify</groupId>
+                        <artifactId>dockerfile-maven-plugin</artifactId>
+                        <version>1.4.13</version>
+                        <executions>
+                            <execution>
+                                <id>coordinator</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>build</goal>
+                                </goals>
+                                <configuration>
+                                    <repository>presto-coordinator</repository>
+                                    <tag>latest</tag>
+                                    <contextDirectory>${project.parent.basedir}/docker/</contextDirectory>
+                                    <dockerfile>${project.parent.basedir}/docker/Dockerfile</dockerfile>
+                                    <buildArgs>
+                                        <PRESTO_VERSION>0.289-SNAPSHOT</PRESTO_VERSION>
+                                    </buildArgs>
+                                    <writeTestMetadata>false</writeTestMetadata>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>worker</id>
+                                <phase>install</phase>
+                                <goals>
+                                    <goal>build</goal>
+                                </goals>
+                                <configuration>
+                                    <repository>presto-worker</repository>
+                                    <tag>latest</tag>
+                                    <contextDirectory>${project.basedir}</contextDirectory>
+                                    <dockerfile>${project.basedir}/scripts/dockerfiles/prestissimo-runtime.dockerfile</dockerfile>
+                                    <buildArgs>
+                                        <BUILD_TYPE>Release</BUILD_TYPE>
+                                        <DEPENDENCY_IMAGE>public.ecr.aws/oss-presto/presto-native-dependency:latest</DEPENDENCY_IMAGE>
+                                        <EXTRA_CMAKE_FLAGS>-DPRESTO_ENABLE_PARQUET=ON -DPRESTO_ENABLE_S3=ON</EXTRA_CMAKE_FLAGS>
```

Comment:
None of the tests use Parquet or S3. We should enable these later.

In any case if we want to use s3 we would need to also bring up a minio server atleast. How are we doing that ?